lib/config/global.ts

```diff
@@ -29,6 +28,52 @@ export class GlobalConfig {
     'gitTimeout',
     'platform',
     'endpoint',
+    'autodiscover',
+    'autodiscoverFilter',
+    'autodiscoverTopics',
```

Comment:
Will ones like this still be deleted from config at an early stage, and not copied available via `GlobalConfig.get()`?

Reply:
These options will now be available via `GlobalConfig.get()`. 

They weren't available  previously, because they were present in `GlobalConfig.OPTIONS` and were ignored by `GlobalConfig,set()`. The logic thought these weren't global config options., therefore instead of setting it to `GlobalConfig` it just returned it  ie. it got added to the repoConfig (`renovate.json`).

This happens here: 
https://github.com/renovatebot/renovate/blob/effb8b605b87884fee50404df35b4b735c700364/lib/workers/repository/index.ts#L41-L43

Reply:
I don't think we ever want to GlobalConfig.get() them. So the question is whether this is a bad thing or not.